### ISSUE TITLE
max-len rule doesn't respect eslint-disable directive

### ISSUE THREAD

**Issue by zxqfox** [eslint/eslint#8778]
**Tell us about your environment**

* **ESLint Version:** v3.19 (feels like any)
* **Node Version:** v4.5.0 (-//-)
* **npm Version:** v2.15.11 (-//-)

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

```
module.exports = {
  rules: {
    'max-len': ["error", 20]
  }
};
```

**What did you do? Please include the actual source code causing the issue.**

No error:
```js
/* eslint-disable */var q0123456789 = 0x123456789/* eslint-enable */;
```

Error:
```js
    /* eslint-disable */var q0123456789 = 0x123456789/* eslint-enable */;
```

**What did you expect to happen?**

No error.

**What actually happened? Please include the actual, raw output from ESLint.**

Max-len rule reporting an error.

**Comment by nzakas** [comment]
Unfortunately, it looks like there wasn't enough interest from the team or community to implement this change. While we wish we'd be able to accommodate everyone's requests, we do need to prioritize. We've found that issues failing to be implemented after 90 days tend to never be implemented, and as such, we [close those issues](https://eslint.org/docs/maintainer-guide/issues#when-to-close-an-issue). This doesn't mean the idea isn't interesting or useful, just that it's not something the team can commit to.
